django/contrib/admin/widgets.py

```diff
@@ -150,6 +150,8 @@ def get_context(self, name, value, attrs):
             context['link_title'] = _('Lookup')
             # The JavaScript code looks for this class.
             context['widget']['attrs'].setdefault('class', 'vForeignKeyRawIdAdminField')
+            if isinstance(rel_to._meta.pk, UUIDField):
+                context['widget']['attrs']['class'] += ' ' + 'vUUIDField'
```

Comment:
I think a test could be added to show when this class is rendered.

Reply:
I just tried adding a test for this but honestly am spending far too long on it; I'm not familiar enough with the way the test suite is rendering those admin fields.